InstallSpinnaker.sh

```diff
@@ -245,6 +262,92 @@ function set_defaults_from_environ() {
       echo "No providers are enabled by default."
   fi
 }
+
+function add_apt_repositories() {
+  # Redis
+  # https://launchpad.net/~chris-lea/+archive/ubuntu/redis-server
+  add-apt-repository -y ppa:chris-lea/redis-server
```

Comment:
Arent you still adding these repositories?
You cant do that in DOWNLOAD=true mode.